src/plugins/intel_npu/src/plugin/npuw/just_sync_infer_request.cpp

```diff
@@ -193,8 +191,10 @@ ov::npuw::JustInferRequest::JustInferRequest(const std::shared_ptr<ov::npuw::Com
             const auto& iport = func_desc.compiled_model->inputs()[closure_param_id];
 
             // No update required to this tensor in runtime - so it can be set only once
-            if (!comp_model_desc.update_required[cidx]) {
-                request->set_tensor(iport, ov::get_tensor_impl(wbank->get(closure, *func_desc.device_it)));
+            // Will be utilized when there is no FOLDing
+            if (!m_npuw_model->m_update_required) {
```

Comment:
Previously, what was the reason to decide this per-tensor?

Reply:
it was in your previous review. `NPUW_FOLD` sets this flag